elastalert/alerts.py

```diff
@@ -1205,7 +1205,7 @@ def alert(self, matches):
                     if self.slack_ca_certs:
                         verify = self.slack_ca_certs
                     else:
-                        verify = self.slack_ignore_ssl_errors
+                        verify = not self.slack_ignore_ssl_errors
```

Comment:
If slack_ignore_ssl_errors = True, then you don't want to validate.

https://requests.readthedocs.io/en/master/user/advanced/#ssl-cert-verification